src/api/accessToken.ts

```diff
@@ -7,14 +7,16 @@ const ENDPOINT_PATH = "/api/backend/v1/access_token"
 export type CreateAccessTokenRequest = {
     userId: string
     durationInMinutes: number
+    activeOrgId?: string
```

Comment:
I would just include `activeOrgId` and `withActiveOrgSupport` should be true if it's truthy